solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -897,6 +899,20 @@ private void reloadSecurityProperties() {
     initializeAuditloggerPlugin((Map<String, Object>) securityConfig.getData().get("auditlogging"));
   }
 
+  private void warnUsersIfSecurityDisabled() {
+    if (authenticationPlugin == null || authorizationPlugin == null) {
+      log.warn("Not all security plugins configured!  authentication={} authorization={}.  Solr is only as secure as " +
+          "you make it. Consider configuring authentication/authorization before exposing Solr to users internal or " +
+          "external.  See https://lucene.apache.org/solr/guide/authentication-and-authorization-plugins.html for more info",
```

Comment:
Point to securing-solr parent page instead?

Reply:
Sure.

Reply:
should we use an s.apache.org shortened link for this?